openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -889,6 +893,8 @@ The JWT MUST contain the following elements:
 
 The Credential Issuer MUST validate that the JWT used as a proof is actually signed by a key identified in the JOSE Header.
 
+If an `attestation` is provided, successfully validated by the Credential Issuer, and the JWT used as proof was signed by one of the attested keys, the Credential Issuer SHOULD return a Credential for each of the keys provided in the `attested_keys` claim of the attestation.
```

Comment:
Could the guidance be more general here e.g

```suggestion
The Credential Issuer SHOULD return a Credential for each of the keys provided in the `attested_keys` claim of the attestation.
```

Im not sure we need to caveat the normative statement to only imply this is the behaviour when the attestation is validated and or signed by one of the attested keys do we? Or perhaps I'm missing something, is there a case where we cont intent to return a credential for each of the keys provided in the `attested_keys` claim of the attestation.

Reply:
We could just remove this sentence as we restate the normative requirement here https://github.com/openid/OpenID4VCI/pull/389/files#diff-1f424614b35a9899813079f1b1f6218631a2aedd993368ccb89bb81a9eda0289R1011

Reply:
lgtm